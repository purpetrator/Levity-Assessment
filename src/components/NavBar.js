import React from "react";

export function Nav() {
  return (
    <nav className="navbar">
      <div className="container-fluid">
        <span className="navbar-brand mb-0 h1">Github Repository Search</span>
      </div>
    </nav>
  );
}

export default Nav;
