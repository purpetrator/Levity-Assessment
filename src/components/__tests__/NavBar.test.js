import React from "react";
import TestRenderer from "react-test-renderer";
import { Nav } from "../NavBar";

it("successful renders the component", () => {
    const component = TestRenderer.create(
        <Nav />
    );

    expect(component).toMatchSnapshot();
});
