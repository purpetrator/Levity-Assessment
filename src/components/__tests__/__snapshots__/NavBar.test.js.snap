// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`successful renders the component 1`] = `
<nav
  className="navbar"
>
  <div
    className="container-fluid"
  >
    <span
      className="navbar-brand mb-0 h1"
    >
      Github Repository Search
    </span>
  </div>
</nav>
`;
